README.md

```diff
@@ -16,6 +16,13 @@ cd csvtofhir
 # create virtual environment and create an "editable" install
 python3 -m venv venv && source venv/bin/activate && python3 -m pip install --upgrade pip setuptools
 python3 -m pip install -e .[dev]
+
+# From the venv: install the packages
+python setup.py install
+
+# From the venv: build
+python setup.py build
```

Comment:
This step isn't required as part of the setup. In other words if we run through the preceding steps we will have an "editable" install in our virtual environment which allows us to execute tests or use the `csvtofhir` CLI.

If we would like to include this command, it should be in a separate section on building a distribution which in our case would be a `wheel file`.